python/pyspark/sql/tests/connect/test_parity_types.py

```diff
@@ -86,6 +86,9 @@ def test_rdd_with_udt(self):
     def test_udt(self):
         super().test_udt()
 
+    def test_collated_string(self):
```

Comment:
We can actually just remove this. It will automatically run the tests (by inheritance)